dimod/discrete/discrete_quadratic_model.py

```diff
@@ -577,7 +598,7 @@ def set_quadratic_case(self, u, u_case, v, v_case, bias):
             self.variables.index(v), v_case,
             bias)
 
-    def _to_file_numpy(self, file, compress):
+    def _to_file_numpy(self, file, compress, save_offset=False):
```

Comment:
IMO always save the offset in `.to_file()`, but minor increment the version. to account for that.

Reply:
So just update the second element of: https://github.com/dwavesystems/dimod/blob/a4bca1031f32d4d12235170704c6dab4144986bf/dimod/discrete/discrete_quadratic_model.py#L41 to 1?

Reply:
yup

Reply:
Is your suggestion to set this to arg to `True` or to do away with it?